src/org/freedesktop/gstreamer/glib/GObject.java

```diff
@@ -22,40 +22,24 @@
  */
 package org.freedesktop.gstreamer.glib;
 
-import static org.freedesktop.gstreamer.lowlevel.GObjectAPI.GOBJECT_API;
```

Comment:
The Import-Reordering happens automatically in IntelliJ and I'd have to reconfigure it to avoid it. If this is important to you, I can do this, but it would be simpler to just let IntelliJ format the File. This would also fix a lot of the Whitespasce-Issues that are in the Code and that I did not commit this time. How do you think about this?

Reply:
The conversation in #163 still stands!  Tell IntelliJ to leave it alone, or use a better IDE. :wink: